app/admin/dashboard.rb

```diff
@@ -0,0 +1,41 @@
+ActiveAdmin.register_page 'Dashboard' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.